<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCharacterForm</class>
 <widget class="QDialog" name="CreateCharacterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="nameLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit">
         <property name="maximumSize">
          <size>
           <width>500000</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="classLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="classLabel">
         <property name="text">
          <string>Class</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="classComboBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="raceLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="raceLabel">
         <property name="text">
          <string>Race</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="raceComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="healthPointLayout">
       <item row="0" column="2">
        <widget class="QLineEdit" name="healthPointEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="healthPointLabel">
         <property name="text">
          <string>HP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="dvLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="dvLabel">
         <property name="text">
          <string>DV</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dvEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="defLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="defLabel">
         <property name="text">
          <string>DEF</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="defEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>135</width>
       <height>214</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="2">
    <layout class="QGridLayout" name="attackLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="attackLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Attack</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="modifLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Modificator</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="contactLabel">
       <property name="text">
        <string>Contact</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="attackContactEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="distanceLabel">
       <property name="text">
        <string>Distance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="attackDistanceEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="magicalLabel">
       <property name="text">
        <string>Magical</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="attackMagicalEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" rowspan="2" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="caracLayoutEmptyLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="caracLayoutValLabel">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="caracLayoutModLabel">
       <property name="text">
        <string>Modificator</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="forRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="forLabel">
       <property name="text">
        <string>FOR</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="createcharacterCaracLineEdit" name="forEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="forModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dexLabel">
       <property name="text">
        <string>DEX</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="createcharacterCaracLineEdit" name="dexEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="dexRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="dexModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="conLabel">
       <property name="text">
        <string>CON</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="createcharacterCaracLineEdit" name="conEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="conRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="conModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="intLabel">
       <property name="text">
        <string>INT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="createcharacterCaracLineEdit" name="intEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="intRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="intModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="sagLabel">
       <property name="text">
        <string>SAG</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="createcharacterCaracLineEdit" name="sagEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="sagRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="sagModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="chaLabel">
       <property name="text">
        <string>CHA</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="createcharacterCaracLineEdit" name="chaEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can drag and drop this value to another place of your choice.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="chaRaceEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="chaModEdit">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>createcharacterCaracLineEdit</class>
   <extends>QLineEdit</extends>
   <header>createcharactercaraclineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>forEdit</tabstop>
  <tabstop>dexEdit</tabstop>
  <tabstop>conEdit</tabstop>
  <tabstop>intEdit</tabstop>
  <tabstop>sagEdit</tabstop>
  <tabstop>chaEdit</tabstop>
  <tabstop>defEdit</tabstop>
  <tabstop>attackContactEdit</tabstop>
  <tabstop>attackDistanceEdit</tabstop>
  <tabstop>attackMagicalEdit</tabstop>
  <tabstop>forRaceEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>forModEdit</tabstop>
  <tabstop>dexRaceEdit</tabstop>
  <tabstop>dexModEdit</tabstop>
  <tabstop>classComboBox</tabstop>
  <tabstop>conRaceEdit</tabstop>
  <tabstop>conModEdit</tabstop>
  <tabstop>raceComboBox</tabstop>
  <tabstop>intRaceEdit</tabstop>
  <tabstop>intModEdit</tabstop>
  <tabstop>healthPointEdit</tabstop>
  <tabstop>sagRaceEdit</tabstop>
  <tabstop>sagModEdit</tabstop>
  <tabstop>dvEdit</tabstop>
  <tabstop>chaRaceEdit</tabstop>
  <tabstop>chaModEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateCharacterForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateCharacterForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
